llvm/include/llvm/ProfileData/MemProf.h

```diff
@@ -342,6 +342,28 @@ using CallStackId = uint64_t;
 // A type representing the index into the call stack array.
 using LinearCallStackId = uint32_t;
 
+// Holds call site information with indexed frame contents.
+struct IndexedCallSiteInfo {
+  // The call stack ID for this call site
+  CallStackId CSId = 0;
+  // The GUIDs of the callees at this call site
+  std::vector<GlobalValue::GUID> CalleeGuids;
```

Comment:
Will it be straightforward to change this later if we decide to dedup callee guids across different records and use a 32-bit linear id?

Reply:
I'm hoping to defer decisions like this one to the point where I make the format changes (after implementing the actual logic for VP annotation in MemProfUse). I don't expect it will be too difficult though.

Reply:
Resolving.